lib/diagnostic.rb

```diff
@@ -13,28 +13,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.odd?
```

Comment:
Not quite. Fixnums have a `.odd?` method; arrays don't. Here's one solution:
`numbers.any? { |num| num.odd? }` 
(alternatively, `numbers.any?(&:odd?)`)